framework/wazuh/core/pyDaemonModule.py

```diff
@@ -80,6 +80,25 @@ def create_pid(name: str, pid: int):
             os.chmod(filename, 0o640)
         except OSError as e:
             raise WazuhInternalError(3002, str(e))
+    
+
+def get_parent_pid(name: str) -> int:
+    """Given the name of a daemon, return its parent ID.
+    
+    Parameters
+    ----------
+    name : str
+        Daemon name.
+    
+    Returns
+    -------
+    pids : int
+        Parent process ID.
+    """
+    regex = rf'{name}*-(\d+).pid'
+    for pid_file in os.listdir(common.OSSEC_PIDFILE_PATH):
```

Comment:
Since we are using the `OSSEC_PIDFILE_PATH` constant, we could rename it to `WAZUH_SERVER_PIDFILE_PATH` or similar

Reply:
We decided to perform these changes in a separate issue because of the amount of code involved.